bibliography-sage.bib

```diff
@@ -1837,12 +1837,20 @@ @article{FiteEtAl2012
   volume       = {148},
 }
 
-@misc{FloriRandriam2011,
+@article{FloriRandriam2011,
   author       = {Flori, Jean-Pierre and Randriam, Hugues},
-  date         = {2011},
-  howpublished = {Cryptology ePrint Archive: Report 2011/245},
-  note         = {http://eprint.iacr.org/2011/245},
   title        = {On the Number of Carries Occuring in an Addition $\mod 2^k-1$},
+  volume       = {12},
+  ISSN         = {1867-0652},
+  url          = {http://dx.doi.org/10.1515/integers-2011-0124},
+  DOI          = {10.1515/integers-2011-0124},
+  number       = {4},
+  journal      = {Integers},
+  publisher    = {Walter de Gruyter GmbH},
+  year         = {2012},
+  month        = {Jan}
```

Comment:
btw, in order to spot the problem, I ran biber to format the bib file (that's the formatter on cocalc for bib files). it reported: 

```
ERROR - BibTeX subsystem: /tmp/qKkIbDXkQB/tmp-106KYl7WqWHM7F.tmp_33083.utf8, line 1852, syntax error: found "date", expected end of entry ("}" or ")") (skipping to next "@")
INFO - WARNINGS: 3
INFO - ERRORS: 1
```

I'll take care